nexus/src/db/datastore.rs

```diff
@@ -239,7 +239,7 @@ impl DataStore {
                 .do_update()
                 .set((
                     dsl::time_modified.eq(Utc::now()),
-                    dsl::pool_id.eq(excluded(dsl::id)),
+                    dsl::pool_id.eq(excluded(dsl::pool_id)),
```

Comment:
I don't *think* this was the manifestation of the problem, but this was definitely a bug.